src/util/removeIrrelevantAttribute.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
May not be worth it at this time, but could this be abstracted in a way that it could remove an arbitrary attribute, instead of only `irrelevant` ones?

Reply:
Could be, but I can't think of any other attributes we would want to remove right now